.grabl/automation.yml

```diff
@@ -113,7 +108,7 @@ build:
         bazel test //test/behaviour/graql/language/define:test-core --test_output=streamed --jobs=1
         bazel test //test/behaviour/graql/language/undefine:checkstyle --test_output=streamed
         bazel test //test/behaviour/graql/language/undefine:test-core --test_output=streamed --jobs=1
-
+    # TODO: turn on cluster tests once cluster tests pass for definable
```

Comment:
Unrelated to this PR, but do you know why define test fails on cluster?

Reply:
I have not looked into it.